src/Microsoft.VisualStudio.Web.CodeGeneration.Design/ToolCommandLineHelper.cs

```diff
@@ -35,11 +36,12 @@ internal static string[] FilterExecutorArguments(string[] args)
             List<string> filteredArgs = new List<string>();
             for (int i = 0; i < args.Length; i++)
             {
-                switch (args[i])
+                switch (args[i].ToLower())
```

Comment:
Generally, when changing case for a switch, it's preferred to use `ToUpperInvariant`